docs/docs-components/SandpackExample.js

```diff
@@ -118,80 +146,149 @@ export default function SandpackExample({
   hideEditor?: boolean,
 |}): Node {
   const { files } = useLocalFiles();
+  const { showDevelopmentEditor } = useDocsConfig();
+
+  const scope = { ...gestalt, DatePicker };
+
+  const regex = /import (.|\n)*('gestalt'|'react');/gi;
+  const regexReact = /(Fragment|useState|useRef|useEffect)/gi;
+
+  const codeFileCleaned = code
+    ?.toString()
+    .replace(regex, '')
+    .replace('export default', '')
+    .replace(regexReact, 'React.$&');
 
-  // Based on
-  // https://github.com/codesandbox/sandpack/blob/53811bb4fdfb66ea95b9881ff18c93307f12ce0d/sandpack-react/src/presets/Sandpack.tsx#L67
   return (
-    <SandpackProvider
-      template="react"
-      files={{
-        '/styles.css': {
-          code: `@import "gestalt/dist/gestalt.css";
+    <Fragment>
+      {showDevelopmentEditor ? (
```

Comment:
This just seems like a massive conditional. Maybe consider breaking to a separate file or making it's own component, or rendering in a different function, so it can be followed a little easier